apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -90,6 +91,18 @@ marionette('Dialer > Keypad', function() {
       return (number.getAttribute('value') === '');
     });
   }
+  
+  test('Add a contact from dialer', function() {
+        
+        var typeNumber = subject.keypadView.typePhoneNumber(12345);
+        var addToContactsActivities = typeNumber.addToContacts();
+        var newContact = addToContactsActivities.pickCreateNew();
+        var dialerOptions = newContact.createNewContactAndReturnToDialer();
```

Comment:
'Name' is hard coded in one of the view, which is problematic. If one day, we had to change the name of the contact to "First Last", we'd have to update both the test and the view.

The test should remain the owner of the data, and pass it to the view. In other words, I'd recommend:
```js
var contactName = 'Name';
var dialer = newContact.createNewContactAndReturnToDialer(contactName);
assert.strictEqual(contactCreated, contactName);
```